synfig-studio/plugins/lottie-exporter/shapes/gFill.py

```diff
@@ -12,9 +12,83 @@
 from common.Gradient import Gradient
 sys.path.append("..")
 
+
+def gen_radial_gradient(lottie, layer, idx):
+    """
+    Generates the dictionary correspnding to shapes/gFill.json but with radial gradient
+
+    Args:
+    """
+    index = Count()
+    lottie["ty"] = "gf"
+    lottie["r"] = 1    # Don't know it's meaning yet, but works without this also
+    lottie["o"] = {}   # Opacity of the gradient layer
+    lottie["nm"] = layer.get_description()
+    lottie["t"] = 2    # 2 means radial gradient layer
+    lottie["s"] = {}   # Starting point of gradient
+    lottie["e"] = {}   # Ending point of gradient
+    lottie["g"] = {}   # Gradient information is stored here
+
+    # Color Opacity
+    opacity = layer.get_param("amount").get()
+    is_animate = is_animated(opacity[0])
+    if is_animate == 2:
+        # Telling the function that this is for opacity
+        opacity[0].attrib['type'] = 'opacity'
+        gen_value_Keyframed(lottie["o"], opacity[0], index.inc())
+
+    else:
+        if is_animate == 0:
```

Comment:
Here too (constant)

Reply:
same thing here also, as mentioned in my above comment